modules/camera/config.py

```diff
@@ -1,4 +1,8 @@
+import sys
+
 def can_build(env, platform):
```

Comment:
To match style guide:
```suggestion
import sys


def can_build(env, platform):
```

Reply:
Though I would suggest moving the `import sys` inside `can_build`.